input/vlct/dual_energy_shock_tube/compute_align_size.py

```diff
@@ -0,0 +1,94 @@
+import sys
+import math
+
+import numpy as np
+
+# Determining the velocity:
+# In the normal case, we use a domain with stretching from 0 to 1 with 256
+# cells; this gives a dx = 1. / 256 . We also know that the interval of time
+# that we allow the cloud to evolve over is dt = 0.25 . Additionally, we can
+# compute the sound speed in the hottest gas of the initial conditions using
+# cs = sqrt (gamma * p / rho); canonically, gamma=1.4, p = 1, and rho = 1.
+#
+# If we are targetting some Mhat then we can determine the closest velocity
+# that leads to a translation over an integral number of cells over dt. The
+# number of cells is given by:
+#    Ncells = floor(cs * Mhat / (dx / dt) + 0.5)
+# We could further require that Ncells be divisible by 4:
+#    Ncells = floor(floor(cs * Mhat / (dx / dt) + 0.5)/4 + 0.5)
+# Once we know Ncells, we can easily compute the additional velocity component
+#    velocity = Ncells * dx / dt
+
+# the interval over which the problem usually evolves (without the
+# transformation)
+_DEFAULT_XLIM = (0.,1.0)
+
+# resolution of the cells
+_ORIG_CELLS = 128
+
+# time over which the system evolves
+_DT = 0.25
+
+# designates that x-axis must have a total number of cells divisable by 4:
+_TOTCELLS_REQ_FACTOR = 4
+
+# adiabatic index
+_GAMMA = 1.4
```

Comment:
Gamma is 5/3 in the input file. Is it different here on purpose?

Reply:
The value of gamma is from `input/vlct/vlct.incl` is overwritten in `input/vlct/dual_energy_shock_tube/initial_sod_shock_tube.incl`. I'll add a comment elaborating on this on line 35 of this file